pathfinder/tests/fixtures/accounts.py

```diff
@@ -14,6 +14,8 @@ def faucet_private_key(request) -> str:
     if is_hex(private_key):
         assert len(remove_0x_prefix(private_key)) == 64
         return private_key
+    else:
```

Comment:
just raise NotImplementedError and let the user deal with it

Reply:
Fixed